packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -1925,24 +1925,7 @@ interface IBasePickerStyles {
 }
 
 // @public (undocumented)
-interface IBasePickerSuggestionsProps {
-  className?: string;
-  forceResolveText?: string;
-  loadingText?: string;
-  mostRecentlyUsedHeaderText?: string;
-  noResultsFoundText?: string;
-  onRenderNoResultFound?: IRenderFunction<void>;
-  resultsFooter?: () => JSX.Element;
-  resultsFooterFull?: () => JSX.Element;
-  resultsMaximumNumber?: number;
-  searchForMoreText?: string;
-  searchingText?: string;
-  showRemoveButtons?: boolean;
-  suggestionsAvailableAlertText?: string;
-  suggestionsClassName?: string;
-  suggestionsContainerAriaLabel?: string;
-  suggestionsHeaderText?: string;
-  suggestionsItemClassName?: string;
+interface IBasePickerSuggestionsProps<T = any> extends Pick<ISuggestionsProps<TextDecoderOptions>, 'onRenderNoResultFound' | 'suggestionsHeaderText' | 'mostRecentlyUsedHeaderText' | 'noResultsFoundText' | 'className' | 'suggestionsClassName' | 'suggestionsItemClassName' | 'searchForMoreText' | 'forceResolveText' | 'loadingText' | 'searchingText' | 'resultsFooterFull' | 'resultsFooter' | 'resultsMaximumNumber' | 'showRemoveButtons' | 'suggestionsAvailableAlertText' | 'suggestionsContainerAriaLabel'> {

```

Comment:
I'm curious if you've been able to verify this truly appears as a 100% compatible API change to consumers?

Reply:
Manually & by checking against API consumers in examples.

But like I said in the description, it diverges for resultsFooterFull and resultsFooter.